llvm/test/CodeGen/AMDGPU/GlobalISel/lround.ll

```diff
@@ -0,0 +1,551 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -global-isel -mtriple=amdgcn -mcpu=gfx900 < %s | FileCheck -check-prefixes=GCN,GFX9 %s
```

Comment:
Just add global isel run lines to the test in the AMDGPU directory instead of duplicating the whole file